docs/trees/guide.md

```diff
@@ -37,16 +37,6 @@ All code examples assume you have the following import
 import scala.meta._
 ```
 
-### Ammonite REPL
-
-A great way to experiment with Scalameta is to use the
-[Ammonite REPL](http://ammonite.io/#Ammonite-REPL).
-
-```scala
-// Ammonite REPL
-import $ivy.`org.scalameta::scalameta:@VERSION@`, scala.meta._
```

Comment:
What do you think about recommending to use a hardcoded version of an older Scalameta release?

```
❯ amm
Loading...
Welcome to the Ammonite Repl 2.3.8-32-64308dc3 (Scala 2.13.4 Java 1.8.0_272)
@ q"a"
res1: Term.Name = Term.Name(value = "a")
```

We can add a note that newer versions don't work yet and link to the issue?

Reply:
The import line is missing from the code above
```
❯ amm
Loading...
Welcome to the Ammonite Repl 2.3.8-32-64308dc3 (Scala 2.13.4 Java 1.8.0_272)
@ import $ivy.`org.scalameta::scalameta:4.4.6`, scala.meta._
qimport $ivy.$                               , scala.meta._

@ q"a"
res1: Term.Name = Term.Name(value = "a")
```

Reply:
I added the `--thin` optino as suggested by @alexarchambault Thanks!